internal/controller/controllers/installenv_controller.go

```diff
@@ -127,6 +151,9 @@ func (r *InstallEnvReconciler) ensureISO(ctx context.Context, installEnv *adiiov
 		return ctrl.Result{Requeue: Requeue}, nil
 	}
 
+	// Check for updates from user, compare spec and trigger ClusterDeploymentsReconciler if found relevant
+	r.notifyClusterUpdatesIfNeeded(installEnv, c)
```

Comment:
if there are any updates that need to be done then this reconcile loop should end here no? no need to generate the ISO if it's going to be generated again after cluster is updated.

Reply:
Done.